package.json

```diff
@@ -1507,6 +1507,11 @@
             "default": true,
             "description": "%configuration.dotnet.projects.enableAutomaticRestore%"
           },
+          "dotnet.projects.enableFileBasedPrograms": {
+            "type": "boolean",
+            "default": true,
+            "description": "%configuration.dotnet.projects.enableFileBasedPrograms%"
```

Comment:
There's support for marking these as preview:

https://code.visualstudio.com/docs/configure/settings#_feature-lifecycle

Should we mark this as preview, since we know it has limitations for now?

Reply:
Yeah, I will do that. I looked at [grep.app](https://grep.app/search?f.lang=JSON&f.path.pattern=package.json&regexp=true&q=%22tags%22%3A+%5C%5B%5Cs%2B%22preview%22) for examples. I think the latest commit does the job.